Source/WebKit/UIProcess/Cocoa/_WKWarningView.mm

```diff
@@ -119,15 +120,31 @@
     };
 
     switch (item) {
-    case WarningItem::SafeBrowsingBackground:
-        return colorNamed(@"WKSafeBrowsingWarningBackground");
+    case WarningItem::BrowsingWarningBackground:
+        return WTF::switchOn(warningView.warning->data(), [&] (WebKit::BrowsingWarning::SafeBrowsingWarningData) {
```

Comment:
const SafeBrowsingWarningData&

Otherwise a copy constructor is called.  Same for all.

I also don't think we need to capture anything.